kratos/python_scripts/assign_initial_state_process.py

```diff
@@ -0,0 +1,125 @@
+# Importing the Kratos Library
```

Comment:
the name of this file is still wrong, should be `set_initial_state_process`

Reply:
I agree

Reply:
I'll do it